protocol/directory.go

```diff
@@ -70,13 +74,13 @@ func (d *ConiksDirectory) Register(req *RegistrationRequest) (
 		return nil, ErrorDirectory
```

Comment:
Yes, it should be handled in `HandleOps`. I fixed it in 5c4c4b02c0284c33a4dfd3440576c1d65cb7c823.